### ISSUE TITLE
Change name BlendingEditor to Blender

### ISSUE THREAD

**Issue by PatrykSzwer** [Starcounter/Blending#91]
After a couple of meetings with @Starcounter-Jack during my trip, we decided that "BlendingEditor" should be renamed to "Blender".

The main reason behind this decision is that we don't want users to associate BlendingEditor as a required entity for managing Blending feature of Starcounter. BlendingEditor should be considered more as a category of apps, designed for managing blending, rather than one required solution. If someone decides to create his own BlendingEditor - It should be fully possible to do, and we shouldn't deny that with poor naming.

In conclusion: Our "Blender" should be *A* blending editor, not *THE* blending editor.

**Comment by warpech** [comment]
Right now, our biggest concern is that there already is a Blender app: https://en.wikipedia.org/wiki/Blender_(software)

**Comment by warpech** [comment]
A different idea is to call it "Blending Studio".

**Comment by PatrykSzwer** [comment]
Let's invite @Starcounter/sc-apps, @Mackiovello  to the discussion, maybe they will have some suggestions/ideas. Also, let us know what do you think about "Blending Studio" name that Marcin suggested.

**Comment by miyconst** [comment]
I honestly don't think that `BlendingEditor` is a bad name. Yes, it's kind of a name on it's own, but `Blender` is not much better. On the cell phones, we have `Camera`, `Contacts`, `Messages` apps, and no one assumes that those are the only apps you can use for the purposes.

**Comment by un-tone** [comment]
For me, the Blending Provider can be named Blender, not the Blending Editor.
So, for the BE, "Blending Studio" sounds better.

**Comment by Mackiovello** [comment]
I'm with @miyconst on this one. `Blender` is certainly not a better name; it's both more ambiguous and makes you think it has something to do with the `Blender` class.

I think it would be easier to define an appropriate name if we have a clear definition. Now, the definition is: "BlendingEditor is an app used by designers to define and manage view compositions and layouts." (according to the README). To me, that's confusing - why would you manage compositions with a blending editor? 

Also, as I've said before: every time we rename something people have to catch up. This takes more effort then you might think. The reason in the initial post is not enough of a reason to justify the change.

**Comment by Joltrix** [comment]
> Right now, our biggest concern is that there already is a Blender app: https://en.wikipedia.org/wiki/Blender_(software)

I don't think we can think in such a big scope. In that case we would most likely have to rename a lot of apps. like, Timeline, or Notes? What I'm getting at is: when using such basic names like we are doing, we are bound to run into duplicate names online.

So, unless we change our apps names to something like "Starcounter Blender", or "Starblend", or "Polyblend" etc.(...you get the point :P), we will have "conflicts" with other vendors and apps. So, I don't think that's reason enough to avoid the name Blender.

**Comment by Joltrix** [comment]
And regarding the name Blending Studio. Maybe it's just me, but when I hear that name, I think of a "huge" program, similar to Visual Studio or Android Studio.

What about something "easier", like, "View Manager, "Compositions","Composer App","View Assembler" or "View Builder".

**Comment by mmnosek** [comment]
> And regarding the name Blending Studio. Maybe it's just me, but when I hear that name, I think of a "huge" program, similar to Visual Studio or Android Studio, and not really a "micro-app".

This is the thing! This shouldn't be just a micro-app. Its scope is going (and should be) bigger and bigger. It's simply a tool that should allow end users (solution owners) to build final software solution out of independently developed apps. I can imagine wider scope:
* layouts
* theming
* blending
* apps installing
* page titles
* exporting/importing configurations
* internationalization

All you can imagine is needed to build a final solution on the top of Starcounter. I would take it one step forward and name it `Starcounter Solution Studio` or `Starcounter Studio` or something in this fashion.

**Comment by miyconst** [comment]
>And regarding the name Blending Studio. Maybe it's just me, but when I hear that name, I think of a "huge" program, similar to Visual Studio or Android Studio, and not really a "micro-app".

@Joltrix, it's not just you.

>This is the thing! This shouldn't be just a micro-app. ...

Hm, we should not rename dough to be bread, just because it supposed to be bread in the end. Still, I get the point.

>All you can imagine is needed to build a final solution on the top of Starcounter. I would take it one step forward and name it `Starcounter Solution Studio` or `Starcounter Studio` or something in this fashion.

If we go this way, then it should be called `Polyjuice Studio`, since Starcounter is the core product, and Polyjuice is everything about the eco-system and the apps. And you know what? I kind of like *Polyjuice Studio*, this name.

**Comment by habib535** [comment]
`Andriod Studio` provides everything related to "Andriod" i.e. front and backend development, test framework, deployment etc. `Starcounter Studio` will be  confusing since here the scope is UX only. I would go for `Blending Studio`.

> If we go this way, then it should be called Polyjuice Studio

We want to avoid THE blending editor.

**Comment by mmnosek** [comment]
@miyconst @habib535 I get the point about `Starcounter Studio`. As we are back to the `Polyjuice`, `Polyjuice Studio` makes a lot of sense!

> We want to avoid THE blending editor.

So we assume "our" blending editor is not the only one available... I would love to see this problem in the future 😄 If we agree on the fact, that this app is going to be more than it is today, then I would love to see it as `Polyjuice Studio`.

> Also, as I've said before: every time we rename something people have to catch up. This takes more effort then you might think. The reason in the initial post is not enough of a reason to justify the change.

Yes, but on the other hand - how many people are using this nowadays? Better now, not later.
